docker_guide.md

```diff
@@ -0,0 +1,97 @@
+# mozilla-mobile Docker Guide
+TaskCluster runs our automation tasks, such as compilation and unit tests, in a [Docker container](https://en.wikipedia.org/wiki/Docker_(software)). We maintain our own Docker images:
+- [`mozillamobile/focus-android`](https://hub.docker.com/r/mozillamobile/focus-android/)
+- [`mozillamobile/firefox-tv`](https://hub.docker.com/r/mozillamobile/firefox-tv/)
```

Comment:
See: https://hub.docker.com/r/mozillamobile/ - We have some for fenix and android-components too :)